plugins/theme_arc/oomox_plugin.py

```diff
@@ -56,7 +56,7 @@ def __init__(self, transient_for, colorscheme, theme_name, **kwargs):
                 },
                 OPTION_EXPORT_GNOME_SHELL_THEME: {
                     'default': False,
-                    'display_name': _("Generate theme for Gnome _Shell"),
+                    'display_name': _("Generate theme for GNOME _Shell"),
```

Comment:
Theme?

Reply:
According to HIG, "Sentence capitalization" should be used for the checkbox labels.

Reply:
ah, i didn't realized it's not a theme, but export option :-)